python/test/gluon/test_frontend.py

```diff
@@ -1315,3 +1317,34 @@ def test_auto_layout_broadcast():
     # CHECK: [[XBCAST2:%.*]] = tt.broadcast [[XCVT2]]
     # CHECK: arith.muli [[YBCAST2]], [[XBCAST2]] : tensor<16x16xi32, [[BLOCKED]]>
     _ = y * x
+
+
+@gluon.jit
+def amd_mfma_layout_kernel():
+    mfma_layout: ttgl.constexpr = cdna3.AMDMFMALayout(version=4, instr_shape=[32, 32], transposed=True,
+                                                      warps_per_cta=[4, 1], tiles_per_warp=[4, 1], ctas_per_cga=[1, 1],
+                                                      cta_split_num=[1, 1], cta_order=[1, 0])
+
+    layout: ttgl.constexpr = ttgl.BlockedLayout([1, 1], [1, 64], [4, 1], [1, 0])
+
+    x = ttgl.full([128, 32], 0, ttgl.float32, layout)
+    res = ttgl.convert_layout(x, mfma_layout)  # noqa: F841
+
+
+@pytest.mark.parametrize("target", [HIP_TARGET_CDNA3])
```

Comment:
We want to test `CDNA4_TARGET` too (and need to use `version=3` in the above test.